nyx_client/nyx_client/connection.py

```diff
@@ -0,0 +1,52 @@
+# Copyright © 2024 IOTIC LABS LTD. info@iotics.com
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://github.com/Iotic-Labs/nyx-sdk/blob/main/LICENSE
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for connection (3rd party data integrations) definitions in Nyx."""
+
+from dataclasses import dataclass
+from typing import Any
+
+
+@dataclass(frozen=True)
+class Connection:
+    """Represents a remote host on the network."""
+
+    id: str
```

Comment:
Arent comments usually above the property?

Reply:
Yeah this looked odd at first, but all the other classes have it this way round!

Reply:
These are docstrings instead of comments so they have to be below so that sphinx can pick them up